### DISCUSSION TITLE
how can tabs be highlighted?

### DISCUSSION THREAD

**Discussion by amtoine** [dandavison/delta#1449] [Q&A] [answered]
hello there :wave: :yum: 

i see we can change the number of spaces that are used to render tabs, e.g. `--tabs 4` or setting `delta.tabs` in the Git config file :+1: 

however, i was wondering if there was a way to highlight them, not just change the number of spaces :thinking: 
that'd be quite cool to see, for instance in a project where we never want tabs, if a change introduces some :yum: 

> **Note**
> it appears `git --no-pager diff` does not show tabs properly either...

**Comment by th1000s** [comment, marked as answer]
Not yet, but I plan to make `--tabs="<string>"` possible, some work in that direction was done in #1424. It might look like 
[this: a‣······b](https://github.com/dandavison/delta/issues/522#issuecomment-779478726). There is/was also #523.

Related, I'm still pondering if "proper" tabstops (see #1205) are worth the trouble.

**Reply by amtoine** [reply]
oooh i see, thanks for the reply, i'll wait a bit more then :pray: 

i think any kind of highlighting would be fine with me, the biggest thing for me is having something visible :wink:
